hack/functest.sh

```diff
@@ -13,3 +13,16 @@ if [ $? -ne 0 ]; then
 	echo "ERROR: Functest failed."
 	exit 1
 fi
+
+"$OUTDIR_BIN/${GINKGO_TEST_SUITE}_tests" -ginkgo.v \
+        --ocs-registry-image="${CATALOG_FULL_IMAGE_NAME}" \
+	--ocs-subscription-channel="${OCS_SUBSCRIPTION_CHANNEL}" \
+	--upgrade-from-ocs-registry-image="${UPGRADE_FROM_OCS_REGISTRY_IMAGE}" \
+	--upgrade-from-ocs-subscription-channel="${UPGRADE_FROM_OCS_SUBSCRIPTION_CHANNEL}" \
+	--ocs-cluster-uninstall="${OCS_CLUSTER_UNINSTALL}" \
+	--arbiter "$@"
+
+if [ $? -ne 0 ]; then
+	echo "ERROR: Arbiter mode functest failed."
+	exit 1
+fi
```

Comment:
I don't want to do this, we are in effect doubling the length of the functional tests. We need to think about how we want to run this... for now I would just say to not add this.

Reply:
Done